frontend/imports/client/auctionlet.js

```diff
@@ -36,6 +36,7 @@ Template.auctionlet.viewmodel({
     if (singleAuctionlet !== undefined && singleAuction !== undefined) {
       const requiredBid = Auctionlets.calculateRequiredBid(singleAuctionlet.buy_amount, singleAuction.min_increase);
       this.bid(web3.fromWei(requiredBid));
+      singleAuctionlet.unitPrice = singleAuctionlet.buy_amount / singleAuctionlet.sell_amount;
```

Comment:
I would recommend to use BigNumber math for this, instead of relying on Javascript's limited precision.
See `Auctionlets.calculateRequiredBid` for a code example and https://github.com/ethereum/wiki/wiki/JavaScript-API#a-note-on-big-numbers-in-web3js and https://github.com/MikeMcl/bignumber.js/ for API docs.